presto-main/src/main/java/com/facebook/presto/execution/StageExecutionInfo.java

```diff
@@ -99,6 +102,7 @@ public static StageExecutionInfo create(
         long processedInputPositions = 0;
 
         long bufferedDataSize = 0;
+        long outputRowCount = 0;
```

Comment:
What's the difference between outputRowCount and outputPositions? Aren't these the same?

Reply:
It is kind of same and outputPositions is collected from output pipeline while outputRowCount is collected form output buffer. Shall we just use the stats from output buffer instead of pipeline? WDYT?

Reply:
I'm not sure I understand, but it is confusing to have outputPositions and outputRowCount. "Position" term in Presto means "Row Count", hence, these variable names mean the same thing.

Reply:
Per our offline discussion, I will change to use the stats at input side which is more accurate indication of shuffle cost in case of output type like broadcast. Thanks for the suggestion!